include/deal.II/grid/grid_in.h

```diff
@@ -404,6 +406,22 @@ class GridIn
   void
   read_vtk(std::istream &in);
 
+  /**
+   * Read grid data from a unstructured vtu file, saved by deal.II using
+   * GridOut::write_vtu(), with the flag
+   * GridOutFlags::Vtu::serialize_triangulation set to true.
```

Comment:
What about adding something on the line "The routing does not support reading in arbitrary vtu files."